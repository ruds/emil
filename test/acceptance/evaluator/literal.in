123;
-123i;
1.23;
NaN;
-Inf;
#"a";
"ab cd \
\ ef";
"""this is a test.
This is only a test.
In case of emergency \U01F631 "you hear?"
""";
R"===(
raw
strings
can
be
fun
)===";
123 : bigint;
"foo" : string;
