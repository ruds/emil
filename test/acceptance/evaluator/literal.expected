val it = 123 : bigint
val it = -123i : int
val it = 1.23 : float
val it = NaN : float
val it = -Inf : float
val it = #"a" : char
val it = "ab cd  ef" : string
val it = "this is a test.\nThis is only a test.\nIn case of emergency ðŸ˜± \"you hear?\"\n" : string
val it = "\nraw\nstrings\ncan\nbe\nfun\n" : string
val it = 123 : bigint
val it = "foo" : string
